src/components/ActivePage.tsx

```diff
@@ -54,6 +57,37 @@ let touchStartMs: number | undefined;
 let activePanel: HTMLDivElement | undefined;
 let isX = false;
 
+const starIcon = (lineKey: string | undefined) => {
+  if (!lineKey) return;
+
+  if (getStoredLineKeys().indexOf(lineKey) !== -1) {
+    return (
+      <button id="unstar-line" class={"w-3 h-3 p-0.5"} onClick={(e) => clickUnstar(lineKey)}>
```

Comment:
```suggestion
      <button class="w-3 h-3 p-0.5" onClick={(e) => clickUnstar(lineKey)}>
```

ID has to be unique per HTML document, so there's a risk this could be a HTML validation issue. We don't need it for anything anyway, so it can just be removed